app/helpers/accounts_helper.rb

```diff
@@ -37,4 +37,11 @@ def class_mapping(accountable_type)
       "Vehicle" => { text: "text-pink-500", bg: "bg-pink-500", bg_transparent: "bg-pink-500/10", fill: "fill-pink-500", hex: "#F23E94" }
     }.fetch(accountable_type, { text: "text-gray-500", bg: "bg-gray-500", bg_transparent: "bg-gray-500/10", fill: "fill-gray-500", hex: "#737373" })
   end
+
+  def format_accounts_balance(accounts, options = {})
+    accounts.group_by(&:currency)
+            .transform_values { |acc| acc.sum(&:balance_money) }
+            .map { |_currency, balance| balance.to_s }
+            .join(", ")
+  end
```

Comment:
I like this solution!  We're already doing something very similar here:

https://github.com/maybe-finance/maybe/blob/a5212f0f5e8fda87c9aed63e6d4d42424e9a7cd9/app/views/transactions/_date_group.html.erb#L13-L19

Can we consolidate both of these into one?

Reply:
Done!